ui/dashboards/src/components/PanelDrawer/PanelDrawer.tsx

```diff
@@ -72,7 +72,13 @@ export const PanelDrawer = () => {
   }, []);
 
   return (
-    <Drawer isOpen={isOpen} onClose={handleClose} SlideProps={{ onExited: handleExited }} data-testid="panel-editor">
+    <Drawer
+      isOpen={isOpen}
+      onClose={handleClose}
+      PaperProps={{ sx: { width: '1280px' } }}
```

Comment:
Are most drawers going to be 1280px wide? if so maybe we can add this as a prop in the Drawer component directly